src/pybind/mgr/volumes/fs/operations/versions/metadata_manager.py

```diff
@@ -21,6 +21,7 @@
 
 class MetadataManager(object):
     GLOBAL_SECTION = "GLOBAL"
+    METADATA_SUBVOLUME_SECTION = "METADATA_SUBVOLUME"
```

Comment:
`uuid` part is removed to support legacy subvolumes ?

Reply:
We can have only one subvolume metadata section at a time hence we don't need `uuid`.
Maybe we can use `uuid` for storing metadata of snapshots because we can have multiple snap sections in .meta file.